server/zanata-security/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.zanata</groupId>
+    <artifactId>server</artifactId>
+    <version>4.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>zanata-security</artifactId>
+  <name>zanata-security</name>
```

Comment:
Most of our security code will still be in zanata-war. This is just a module to share some common interface and annotation. I am going to put more annotation into this module so that it can be shared. It will probably stay this way forever as long as we need to share code between modules. The zanata-security name came from one of the TODO tag but with a question mark at the end. I just took it and used it. Maybe security-common?